turbopack/crates/turbopack-ecmascript/src/tree_shake/asset.rs

```diff
@@ -40,10 +40,10 @@ impl EcmascriptParsable for EcmascriptModulePartAsset {
         let split_data = split(this.full_module.ident(), this.full_module.source(), parsed);
         Ok(part_of_module(split_data, this.part))
     }
-
     #[turbo_tasks::function]
     async fn parse_original(self: Vc<Self>) -> Result<Vc<ParseResult>> {
```

Comment:
You could avoid `self.await` using this syntax:

```suggestion
    fn parse_original(&self) -> Vc<ParseResult> {
```